lib/openshift/cluster_service_version.rb

```diff
@@ -3,5 +3,26 @@
 module BushSlicer
   class ClusterServiceVersion < ProjectResource
     RESOURCE = "clusterserviceversions"
+  def ready?(user:, quiet: false, cached: false)
+      if cached && props[:raw]
+        res = { instruction: "get cached clusterserviceversion #{name} readiness",
+                response: props[:raw].to_yaml,
+                success: true,
+                exitstatus: 0,
+                parsed: props[:raw]
+        }
+      else
+        res = get(user: user, quiet: quiet)
+      end
+      if res[:success]
```

Comment:
It seems to me that having the resource available means it is ready, correct? Don't we wait for some value to be set before considering the version ready?